plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -29,7 +29,9 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 				}
 			));
 
-			var defaultWidth = size.width * 0.5 - 10;
+			var padding = 40;
+
+			var defaultWidth = size.width * 0.5 - padding;
```

Comment:
Is this `padding` var being used elsewhere?  If not I'd avoid creating the variable

Reply:
you're right, it's not being used anywhere, but I'm guessing @mateomustapic added this to avoid a "magic number"

Reply:
Is not 0.5 already a magic number? 😛

Reply:
Well, it translates pretty obviously to "half" so not all that magical, unlike `padding` which is totally arbitrary. You probably wouldn't want to write:

```js
var padding = 40;

var half = 0.5;

var defaultWidth = size.width * half - padding;
```

Reply:
> Is not 0.5 already a magic number? stuck_out_tongue

yeah, well you make a point;  but I always read that as "whatever / 2"

x / 2 == x * 0.5

Reply:
Sorry I hadn't seen @wincent 's comment before adding mine: now that I've read it, I agree on avoiding **those** magic numbers everywhere